react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -108,31 +108,27 @@
   // End bold icon hacks
 }
 
-.smallDeviceOnly {
-  @media @desktop {
-    display: none;
-  }
-}
-
-.aboveSmallDevice {
-  @media @mobile {
-    display: none;
-  }
-}
-
 .firstItemInGroup {
   @media @mobile {
-    padding-top: @grid-row-height * 2;
+    padding-top: @grid-row-height * 4;
     position: relative;
     &::before {
```

Comment:
I wonder why we're not just doing something like that 
```
    padding-top: @grid-row-height * 2;
    margin-top: @grid-row-height * 2;
    border-top: 1px solid fade(@sk-white, 20%);
```
instead of having all that ::before 🤔

Reply:
I believe we switched to `::before` as borders take space and put the grid off alignment?